app/frontend/components/taskList/taskRow.tsx

```diff
@@ -0,0 +1,76 @@
+import { ErrorMessage } from '@hookform/error-message'
+import { yupResolver } from '@hookform/resolvers/yup'
+import { useForm } from 'react-hook-form'
+import { TaskFragment, TaskInput, useTaskUpdateMutation } from '../../generated/graphql'
+import { InputField } from '../inputField/inputField'
+import { taskInputSchema } from '../taskDetailsModal'
+
+interface TaskRowProps {
+  task: TaskFragment
+}
+
+type TaskRowFormData = Pick<TaskInput, 'title'>
+
+export const TaskRow = ({ task }: TaskRowProps) => {
+  const [{ fetching }, taskUpdate] = useTaskUpdateMutation()
+  const {
+    register,
+    handleSubmit,
+    reset,
+    formState: { errors },
+  } = useForm<TaskRowFormData>({
+    mode: 'onChange',
+    defaultValues: {
+      title: task.title,
+    },
+    resolver: yupResolver(taskInputSchema),
+  })
+
+  const handleSubmitTask = async (taskData: TaskRowFormData) => {
+    try {
+      const result = await taskUpdate({
+        id: task.id,
+        data: {
+          projectId: task.project.id,
+          title: taskData.title,
+        },
+      })
+      if (result.error) {
+        throw new Error(`GraphQL Error ${result.error}`)
+      }
+      reset()
+    } catch (error) {
+      // eslint-disable-next-line no-console
+      console.error(error)
+    }
+  }
+
+  return (
+    <div className="flex w-[500px]">
```

Comment:
I would prefer min/max width.
While we don't have a responsive design, I think we should not use fixed widths where we can.